src/modules/education/dto/create-education.dto.ts

```diff
@@ -0,0 +1,27 @@
+import { IsBoolean, IsDate, IsNumber, IsString } from 'class-validator';
```

Comment:
hmm I'm confusing, we dont need education module, education is always part of user update, we don't work with it separatelly